src/meltano/cli/elt.py

```diff
@@ -105,6 +112,11 @@ class ELOptions:
         is_flag=True,
         help="Merges state with that of previous runs.",
     )
+    run_id = click.option(
+        "--run-id",
+        type=UUIDParamType(),
+        help="Use a custom run ID.",
```

Comment:
We should make it clear that a UUID is still required, rather than allowing any alternative ID format

```suggestion
        help="Provide a UUID to use as the run ID.",
```